### PR TITLE
add config for no-response-robot

### PR DESCRIPTION
**Pull request by braver**
This sets up a bot to close "stale" issues and PR's after two weeks automatically. It's a pretty simple and limited bot, but it's a convenient way to set a two week time bomb under issues where the author has gone AWOL. I use it everywhere in SublimeLinter repo's to keep things clean, it's a little time saver. It cannot apply the "timeout" label though, so it does kinda break our protocol in that sense, but overall I'd say it's a win and the "timeout" labels can always be manually applied afterwards.

### CODE REVIEW

**Comment by FichteFoll** [comment]
LGTM, although we should update the automatic comment text to suit our needs, i.e. mention "pull request" and use more concise wording.

FYI, I cannot configure apps for this repository.

**Comment by braver** [comment]
> FYI, I cannot configure apps for this repository.

Me neither, so that's probably just @wbond then?

File: .github/no-response.yml [hunk 1]
```diff
@@ -6,8 +6,5 @@ daysUntilClose: 14
 responseRequiredLabel: "stale"
 # Comment to post when closing an Issue for lack of response. Set to `false` to disable
 closeComment: >
-  This issue has been automatically closed because there has been no response
-  to our request for more information from the original author. With only the
-  information that is currently in the issue, we don't have enough information
-  to take action. Please reach out if you have or find the answers we need so
-  that we can investigate further.
+  This pull request has been automatically closed because there has been no
+  response to our request for information or changes from the original author.
```
Comment (FichteFoll) [inline .github/no-response.yml:10]: Maybe add something like "you can comment to re-open" (if that's true) or "you may of course open a new pull request" to not discourage people that missed the deadline?

**Comment by jrappen** [comment]
@braver Maybe use <https://github.com/actions/stale> instead? For further reactions <https://github.com/actions/github-script> might help as well.

**Comment by braver** [comment]
🤷🏼‍♂️
